src/components/dialog/index.js

```diff
@@ -28,6 +35,10 @@ class Dialog extends BaseComponent {
      * dismiss callback for when clicking on the background
      */
     onDismiss: PropTypes.func,
+    /**
+     * the direction of the swipe to dismiss the dialog (default is 'down')
+     */
+    dismissSwipeDirection: PropTypes.oneOfType(SWIPE_DIRECTIONS),
```

Comment:
I think `oneOfType` is not correct here, you need oneOf, something like that: 
`PropTypes.oneOf(_.values(SWIPE_DIRECTIONS))`

Reply:
What do you mean by not correct? it does work...

Reply:
It works even when sending the string value (i.e. 'up' instead of Dialog.swipeDirections.UP)